src/CONST.js

```diff
@@ -408,7 +408,7 @@ const CONST = {
         FREQUENTLY_USED_EMOJIS: 'expensify_frequentlyUsedEmojis',
     },
     DEFAULT_TIME_ZONE: {automatic: true, selected: 'America/Los_Angeles'},
-    DEFAULT_ACCOUNT_DATA: {error: '', success: '', loading: false},
+    DEFAULT_ACCOUNT_DATA: {error: '', success: '', isLoading: false},
```

Comment:
This was an issue with multiple error fields. At some places in App, we're using `error` as a string and Web-E sends timestamp keyed object as `errors`. I fixed that at almost all places in my [PR](https://github.com/Expensify/App/pull/10244). I think you need to keep this on HOLD till that PR is merged